test/parallel/test-domain-timers.js

```diff
@@ -30,13 +30,19 @@ timeoutd.on('error', common.mustCall(function(e) {
   assert.strictEqual(e.message, 'Timeout UNREFd',
                      'Domain should catch timer error');
   clearTimeout(timeout);
```

Comment:
Should we clear the timeout only on second call?

Reply:
It shouldn't matter. They will all fire during the same loop run. That timeout is just there to make sure it doesn't exit before it gets a chance to enter the event loop.